TerrorConsole/Assets/_Main/Scripts/Objects/PickableObject.cs

```diff
@@ -9,6 +9,7 @@ public class PickableObject : MonoBehaviour
         private void Start()
         {
             _objectName = _objectName == "" ? transform.name : _objectName;
+            //gameObject.SetActive(!Inventory.Source.IsItemInInventory(_objectName)); The object wont appear if it is alreay in inventory -- Commented this since idk if this is necessary
```

Comment:
Instead of this, create a new issue with a good description and add a new label called `maybe` on the backlog.

Reply:
Remove comment after creating the issue.